include/El/core/imports/aluminum.hpp

```diff
@@ -5,10 +5,19 @@
 
 #ifdef HYDROGEN_HAVE_ALUMINUM
 #include <Al.hpp>
+
+#ifdef HYDROGEN_HAVE_NVPROF
+#include "nvToolsExt.h"
+#include "nvToolsExtCuda.h"
+#include "nvToolsExtCudaRt.h"
+#endif // HYDROGEN_HAVE_NVPROF
+
 #endif // HYDROGEN_HAVE_ALUMINUM
 
 namespace El
 {
+// "Real" declaration is in include/El/core/environment/impl.hpp
+extern void break_on_me();
```

Comment:
Is this supposed to be there, or part of debugging code?

Reply:
It's supposed to be there.